projects/components/src/public-api.ts

```diff
@@ -171,10 +171,11 @@ export { SequenceSegment } from './sequence/sequence';
 export * from './sequence/sequence-chart.component';
 export * from './sequence/sequence-chart.module';
 
-// Sheet
-export { SheetOverlayService } from './sheet/overlay/sheet-overlay.service';
-export * from './sheet/sheet.component';
-export * from './sheet/sheet.module';
+// Overlay
```

Comment:
Can you please try to maintain the alphabetical order here when renaming?

Reply:
Okay :). will update